assets/src/domain/blocks/event-attendees/AttendeeListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Fragment } from '@wordpress/element';
+
+import AvatarImage from '../components/AvatarImage';
+
+interface AttendeeListItemProps {
+	attendee: any; // TODO add types for attendee
```

Comment:
does this need updating as part of this PR?

Reply:
Another PR